Shopping_Cart/Utility.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ShoppingCart
+{
+    public class Utility
+    {
+        public static void DisplayMenu()
+        {
+            Console.WriteLine(KeyStore.WelcomeMessage);
+            Console.WriteLine(KeyStore.Seperator);
+            Console.WriteLine(KeyStore.MenuHeader);
+            string line;
+            try
+            {
+                using (StreamReader sr = new StreamReader(KeyStore.InventoryFilePath))
+                {
+                    while ((line = sr.ReadLine()) != null)
+                    {
+                        Console.WriteLine(line);
```

Comment:
Dont write console.writeline here
Create a function called TableDisplay(string row)
Inside the function, split the row into Code, Name, UnitsAvailable, CostPerUnit
Take a fixed value column width and display this as a proper table